### ISSUE TITLE
Add a source/about link

### ISSUE THREAD

**Issue by noraj** [jfurrow/flood#654]
## Summary

The Flood UI lack of a pointer to its source repository.

## Expected Behavior

There should be an *about*/*author*/*github* page or button.

## Current Behavior

There is no *about*/*author*/*github* page or button.

## Possible Solution

One of the two solutions below:

1) Add a *github* icon next to the *notification* icon with a link to https://github.com/jfurrow/flood

![](https://i.imgur.com/yrYggK7.png)

2) Add an *about* tab in the *settings* page.

![](https://i.imgur.com/hXwB076.png)

## Context

This is useful when you spotted a bug and want to quickly open an issue.

This is also useful for people who didn't installed Flood but get it from an out-of-box, so they can know where the UI comes from.

**Comment by noraj** [comment]
I'm not a great fan of using a third party redirecter.
It's possible for people to drop referer from their reverse proxy.
Example for nginx: `proxy_set_header Referer "";`.

**Comment by bbashy** [comment]
@noraj So you think just link directly?

**Comment by noraj** [comment]
Yes, what do you think ?

**Comment by bbashy** [comment]
Will update it. I guess people will copy the link anyway if they're worried about it.

**Comment by noraj** [comment]
They will may be already using a plugin to drop referer in their browser anyway.

**Comment by bbashy** [comment]
Yeah. Also forgot about `rel="noreferrer"` :) I already had that in there.
